index.html

```diff
@@ -111,7 +111,7 @@ <h2>The Less Quick Version</h2>
             This work is licensed under a
             <a
               rel="license"
-              href="http://creativecommons.org/licenses/by/3.0/deed.en_US"
+              href="http://creativecommons.org/licenses/by/3.0/deed.en"
```

Comment:
Let's switch to HTTPS while we're at it 🙂 
```suggestion
              href="https://creativecommons.org/licenses/by/3.0/deed.en"
```